syntax = "proto3";

package protobufs.common.v1;
option go_package = "protobufs/common/v1";

message SolutionFile {
    bool entry = 1;
    string path = 2;
    bytes data = 3;
}